docker/coconut/qemu.dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM ubuntu:24.04
+
+#RUN sed -Ei 's/^# deb-src /deb-src /' /etc/apt/sources.list \
+RUN apt update \
+    && apt upgrade -y \
+    && apt install -y \
+        git \
+        make \
+        cbindgen \
+        curl \
+        gcc \
+        libcunit1 \
+        libcunit1-doc \
+        libcunit1-dev \
+        gettext \
+        python3-venv \
+        ninja-build \
+        bzip2 \
+        libglib2.0-dev \
+        rustc \
+        iasl \
+        build-essential \
+        libglib2.0-dev \
+        libfdt-dev \
+        libpixman-1-dev \
+        zlib1g-dev \
+        libudev-dev \
+        libvdeplug-dev \
+        libslirp-dev \
+        seabios
+
+#RUN apt-get build-dep qemu
+
+ARG QEMU_DATADIR
+
+RUN curl https://sh.rustup.rs -sSf | sh -s -- -y
```

Comment:
Could you merge these two `RUN` commands, and add a comment like:

```
# Clone and build IGVM
```

Reply:
Fixed